doc/modules/metrics.rst

```diff
@@ -93,6 +93,41 @@ is equivalent to :func:`linear_kernel`, only slower.)
       Information Retrieval. Cambridge University Press.
       https://nlp.stanford.edu/IR-book/html/htmledition/the-vector-space-model-for-scoring-1.html
 
+.. _gower_distances:
+
+Gower distances
+-----------------
+The function :func:`gower_distances` computes the distances between the
+observations in X and Y, that may contain combinations of boolean, numerical,
+or categorical attributes, using an implementation of Gower Similarity.
+
+.. math::
+
+    g(x, y) = sum(s(x, y)) / num_of_not_missing_features
```

Comment:
I think you want `s(x_i, y_i)` for corresponding features of x and y. x and y should be notated as `\mathbf{x}` and `\mathbf{y}` I think. and these underscores on the right will break LaTeX compilation if I'm not mistaken. Rather in the narrative docs I'd use TeX like:
 `g(\mathbf{x}, \mathbf{y}) = \frac{\sum_i(s(x_i, y_i))}{|\{i| x_i\text{ or }y_i\text{ is missing}\}|}`.

But I'd also describe this in narrative: "Gower combines a Manhattan (L1) distance for numeric features with Hamming distance for categorical features"